src/index.ts

```diff
@@ -8,64 +8,116 @@ export * from './packages/ThemeGenerator';
 export * from './packages/ThemeGenerator/ThemeResult';
 
 // Atoms
-export { Button, SearchButton, StepperButton } from './components/Buttons';
-export { Callout } from './components/Callout';
-export { CandidateAvatar } from './components/CandidateAvatar';
-export { Chip } from './components/Chip';
-export { ContentPlaceholder } from './components/ContentPlaceholder';
-export { Heading } from './components/Heading';
-export { StringHighlighter, ReactElementHighlighter } from './components/Highlighter';
+export {
+    Button,
+    ButtonProps,
+    SearchButton,
+    SearchButtonProps,
+    StepperButton,
+    StepperButtonProps,
+} from './components/Buttons';
+export { Callout, CalloutProps } from './components/Callout';
+export { CandidateAvatar, CandidateAvatarProps } from './components/CandidateAvatar';
+export { Chip, ChipProps } from './components/Chip';
+export { ContentPlaceholder, ContentPlaceholderProps } from './components/ContentPlaceholder';
+export { Heading, HeadingProps } from './components/Heading';
+export {
+    StringHighlighter,
+    StringHighlighterProps,
+    ReactElementHighlighter,
+    ReactElementHighlighterProps,
```

Comment:
What about other dependencies types, for example, `ReactElementHighlighterTermResult`? the same for other components.